frontend/src/common/utils/api/handler.ts

```diff
@@ -0,0 +1,21 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import readJson from "./readJson";
+
+const getHandler = (path: string) => {
+  const data = readJson(path);
+
+  async function handler(req: NextApiRequest, res: NextApiResponse) {
+    try {
+      const cellTypeId = req.query.cellTypeId;
+      if (data[cellTypeId as keyof typeof data])
+        res.status(200).json(data[cellTypeId as keyof typeof data]);
+      else res.status(204);
```

Comment:
The HTTP 204 No Content success status response code indicates that a request has succeeded, but no content is returned. This is used to indicate (without raising an error) that a cell type doesn't exist in our database for a particular endpoint.